privacyidea/static/components/token/controllers/tokenControllers.js

```diff
@@ -287,6 +287,9 @@ myApp.controller("tokenEnrollController", function ($scope, TokenFactory,
             // because the user might not be allowed to list RADIUS servers
             $scope.getRADIUSIdentifiers();
         }
+        if ($scope.form.type === "certificate") {
+            $scope.getCAConnectors();
```

Comment:
Is it intentionally that you want to reload the CA Connectors each time you change the tokentype?
Before the caconnector list was loaded only once.

Reply:
You're right, we now reload the CA connectors each time we change the tokentype to "Certificate". But before, we tried to load the CA connectors even if the user is not allowed to enroll certificate tokens, and this causes a red error popup :/